### PR TITLE
Irish D5 PR

### PR DESCRIPTION
**Pull request by IrishKnight0**
sorry for being late, great work on this day, it was fun to read

### CODE REVIEW

File: script/Ciel/Day 5/05_01_CIEL05_3E.txt [hunk 1]
```diff
@@ -1106,7 +1106,7 @@ The Executor's sword repels it.
 [sha:ad9c3b95bdf10fb9e4a8dd008a343bc6f050773a]{
 -- Page 642, Offset 12556.
 -- 人智を超越した斬り合いはいつ果てるともなく続いていく。
-The sword fight that has long transcended human comprehension shows no sign of stopping.
+The swordfight that has long transcended human comprehension shows no sign of stopping.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_3E.txt:1109]: "sword fight" is infinitely more common. I do not believe the google dictionary here

File: script/Ciel/Day 5/05_01_CIEL05_4.txt [hunk 2]
```diff
@@ -139,7 +139,7 @@ She's well within her rights to be angry, but now's not the time.
 -- Page 686, Offset 12676.
 -- 「怒るのは後！　ああもう、引っ張りますよ、いいですね!?
 "You can be mad at me later!
- %{i}Ugh%{/i}, that does it, I'm gonna pull you out, okay?!
+ Geez, that does it, I'm gonna pull you out, okay?!
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4.txt:142]: revert

**Review by Avuxo — APPROVED** [review]
nice work

File: script/Ciel/Day 5/05_01_CIEL05_3C.txt [hunk 3]
```diff
@@ -320,7 +320,7 @@ Senpai and Sensei are both good people, you know?! They've saved me so many time
 [sha:ce7500c1c4f941a1d30dc71e36c1d4f2e54fd585]{
 -- Page 512, Offset 12197.
 -- 「な、なによぅ、あいつらの肩ばっかりもって！
-"W-Whaaat, now you're siding with them!
+"W-What?! Now you're siding with them!
```
[the remark this answers is not in the document]
Comment (IrishKnight0) [reply script/Ciel/Day 5/05_01_CIEL05_3C.txt:323]: I only noticed there are many "Whaaat" and "Whaa" instances in this day especially, but I'll revert it

File: script/Ciel/Day 5/05_01_CIEL05_3D.txt [hunk 4]
```diff
@@ -126,7 +126,7 @@ Each of the blades――there must be over a hundred of them in total――have
 -- Page 552, Offset 12291.
 -- 「百本単位の<黒鍵|こっけん>による結界……そう。志貴の言う代行者って、アイツなワケ」
 "A barrier formed of hundreds of Black Keys...
- Yep. This is definitely the work of the Executors you mentioned, Shiki."
+ Yes. This is definitely the work of the Executors you mentioned, Shiki."
```
[the remark this answers is not in the document]
Comment (IrishKnight0) [reply script/Ciel/Day 5/05_01_CIEL05_3D.txt:129]: Hmm you sure? because she sounds serious here

[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Ciel/Day 5/05_01_CIEL05_3D.txt:129]: Regardless, "yep" is arcueid register

File: script/Ciel/Day 5/05_01_CIEL05_3C.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Ciel/Day 5/05_01_CIEL05_3C.txt:323]: take that up with noel

File: script/Ciel/Day 5/05_01_CIEL05_3E.txt [hunk 5]
```diff
@@ -317,7 +317,7 @@ They struggle to keep their footing atop the frozen glacier.
 -- 呼吸さえ死に繋がる極限状態で、氷弾と弾丸がぶつかり合う。
 Under these extreme conditions,
  where even breathing could spell death,
- bullets go head to head against flying shards of ice.
+ bullets go head-to-head against flying shards of ice.
```
Comment (rschlaikjer) [inline script/Ciel/Day 5/05_01_CIEL05_3E.txt:320]: I guess we should add this to lint

File: script/Ciel/Day 5/05_01_CIEL05_4.txt [hunk 6]
```diff
@@ -264,7 +264,7 @@ I'll happily get up close and personal with that bastard one more time if it mea
 [sha:b22b83954e63a1c8ef61012cf79ef404abc1c514]{
 -- Page 692, Offset 12700.
 -- 「とにかく任せて！　一緒にやらせてくれ、先輩！」
-"Look, just leave it to me!
+"In the meantime, just leave it to me!
```
Comment (rschlaikjer) [inline script/Ciel/Day 5/05_01_CIEL05_4.txt:267]: Gotta say I _really_ don't like literal translations of tonikaku. People don't say this!

**Review by rschlaikjer — COMMENTED** [review]
Got here a bit late but couple comments

**Pull request merged**